StringCalcTests/StringCalcTest.cs

```diff
@@ -0,0 +1,159 @@
+﻿using StringCalc;
+
+namespace StringCalcTests
+{
+    public class StringCalcTest
+    {
+        private readonly Calculator _calc;
+
+        public StringCalcTest()
+        {
+            // Arrange
+            _calc = new Calculator();
+        }
```

Comment:
Doing test set-up in constructors can be dangerous if your system under test (your Calculator in this case) carries any sort of state.